src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -117,6 +117,7 @@
   <!-- The implicit collection in an object or collection initializer expression. -->
   <Node Name="BoundObjectOrCollectionValuePlaceholder" Base="BoundValuePlaceholderBase">
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
+    <Field Name="IsNew" Type="bool" Null="NotApplicable"/>
```

Comment:
>IsNew [](start = 17, length = 5)

"IsObjectBeingCreated" or "IsInstanceBeingCreated"?

Reply:
How about "IsNewInstance"?

---
In reply to: [554125889](https://github.com/dotnet/roslyn/pull/50200#discussion_r554125889) [](ancestors = 554125889)

Reply:
> How about "IsNewInstance"?

Sounds fine to me.


---
In reply to: [554191156](https://github.com/dotnet/roslyn/pull/50200#discussion_r554191156) [](ancestors = 554191156,554125889)